holidays/countries/ireland.py

```diff
@@ -66,11 +83,19 @@ def _country_specific(self, year):
         self[date(year, JUN, 1) + rd(weekday=MO)] = "June Bank Holiday"
 
         # Summer bank holiday (first Monday in August)
-        self[date(year, AUG, 1) + rd(weekday=MO)] = "Summer Bank Holiday"
+        self[date(year, AUG, 1) + rd(weekday=MO)] = "August Bank Holiday"
 
         # October Bank Holiday (last Monday in October)
         self[date(year, OCT, 31) + rd(weekday=MO(-1))] = "October Bank Holiday"
 
+        # Christmas Day
+        name = "Christmas Day"
+        self[date(year, DEC, 25)] = name
+        if self.observed and date(year, DEC, 25).weekday() == SAT:
+            self[date(year, DEC, 27)] = name + " (Observed)"
+        elif self.observed and date(year, DEC, 25).weekday() == SUN:
+            self[date(year, DEC, 27)] = name + " (Observed)"
```

Comment:
Is this part correct?
* if 12/25 (SAT) --> 12/27 (MON)
* if 12/25 (SUN) --> 12/27 (TUE)

This can be simplified as:
```python
        if self.observed and date(year, DEC, 25).weekday() in [SAT,SUN]:
            self[date(year, DEC, 27)] = name + " (Observed)"
```

or maybe like:
```python
        if self.observed and date(year, DEC, 25).weekday() in [SAT, SUN]:
            self[date(year, DEC, 27) + rd(weekday=MO)] = name + " (Observed)"
```

Reply:
Thanks, updated in the latest commit